frontend/packages/ceph-storage-plugin/src/components/independent-mode/utils.ts

```diff
@@ -30,6 +65,12 @@ export const checkForIndependentSupport = (csv: ClusterServiceVersionKind): bool
   return independent === 'true';
 };
 
+export const getRequiredKeys = (csv: ClusterServiceVersionKind): { [key: string]: string[] } => {
+  // external.ocs.openshift.io/validation: '{"encodedKeys":["x", "y"], "plainKeys": ["x", "y", "z"]}'
+  const keys = csv.metadata.annotations?.['external.cluster.ocs.openshift.io/validation'] ?? '{}';
```

Comment:
Define this in constant file `'external.cluster.ocs.openshift.io/validation'`
```suggestion
  const keys = getAnnotation()?.['external.cluster.ocs.openshift.io/validation'] ?? '{}';
```

Reply:
It would make sense to put it in a constant only if it were used in at least one more place.

Reply:
The idea is to make it cleaner and put the new annotation in a common place.

Reply:
Using `getAnnotations` sounds reasonable. But it only makes sense to use in a commonplace if it's used in multiple files in which case importing from a commonplace is the right thing to do.